server/controllers/api/videos/index.ts

```diff
@@ -298,6 +302,7 @@ async function updateVideo (req: express.Request, res: express.Response) {
   }
 
   try {
+    let videoInstanceUpdated
```

Comment:
Why?

Reply:
I don't have strong opinion on this change. I'm sure there is a better way. I could simply make the log inside the transaction.

Reply:
Oh, did not see the log was outside! But yes put it just below `federateVideoIfNeeded` (+ logger.info too please)